pyccel/ast/class_defs.py

```diff
@@ -139,6 +139,8 @@
         methods=[
             PyccelFunctionDef('append', func_class = ListAppend,
                 decorators = {}),
+            PyccelFunctionDef('insert', func_class = ListInsert,
+                decorators = {}),
             PyccelFunctionDef('pop', func_class = ListPop,
                 decorators = {}),
             PyccelFunctionDef('clear', func_class = ListClear,
```

Comment:
If you want to list the methods alphabetically, than maybe move `clear` to before `insert`?

Reply:
fixed in 650a33f87857e6ba07ceeb7a3837a2df10272e9d